src/grt/grt-wave_opt_file-parse.adb

```diff
@@ -59,28 +60,28 @@ package body Grt.Wave_Opt_File.Parse is
                         return Boolean;
 
    -- Parse the line where the version is set
-   procedure Parse_Version (Line : String_Access);
+   procedure Parse_Version (Line : String; Line_Pos : Positive);
```

Comment:
Col or Column is fine with me.
The 'Line' meaning is to overloaded: is it the line string or the line number ?  But Col is clear.